contributing-guides/style-guide.zh.md

```diff
@@ -0,0 +1,90 @@
+# 格式指导
+
+当你在为 `tldr` 贡献时，请遵守下面的格式规范。
+
+请注意，下面的规范仅适用于中文翻译的 `tldr` 页面。
+
+## 排版
+
+首先，你的页面应该看起来像这样：
+
+```
+# 命令名称
+
+> 短小精悍的描述。
+> 描述最好只有一行；当然，如果需要，也可以是两行。
+> 更多信息：<https://example.com>。
+
+- 命令描述：
+
+`命令 -选项1 -选项2 -参数1 {{参数的值}}`
+
+- 命令描述：
+
+`命令 -选项1 -选项2`
+```
+
+当你将自己的贡献提交 pull request 时，一个脚本会自动检查你的贡献是否符合上面的格式。
+
+你也可以在提交前在本地测试自己的贡献：
+
+```
+npm install tldr-lint
+tldrl -f {{page.md}}
+```
+
+关于 `tldrl` 的更多使用方法，例如检查批量检查一整个目录的格式，[`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/master/pages/common/tldrl.md) 是你的不二去处！
+
+如果你用 tldr-pages 的 Node.js 客户端，你可以在命令后加 `-f` (`--render`) 来在本地预览自己的页面：
+
+```
+tldr -f {{page.md}}
+```
```

Comment:
```suggestion
```
tldr --render {{page.md}}
```
```

Reply:
Yep, that's not correct. Above it says `` `-f` (`--render``) ``, which should be `-r`